docs/install-config/configure-internal-tls.md

```diff
@@ -17,6 +17,7 @@ Besides, a user can also provide the certs for all components. However, there ar
 
 * First, all certs must be signed by a single unique CA
 * Second, the filename of the internal cert and `CN` field on cert file must follow the convention listed below'
+* Third, because the self signed certificate without SAN was deprecated in Golang 1.5. For more information please refer to [golang 1.5 release notes](https://golang.org/doc/go1.15#commonname) and [this issue](https://github.com/golang/go/issues/24151). As a result, you MUST add the SAN extension to your cert files when generating certs by yourself. Or the harbor instance will not start up normally. The `DNS` name in `SAN` extertion should the same as `CN` field in the table below.
```

Comment:
Wording suggestion:

Third, because the self signed certificate without SAN was deprecated in Golang 1.5, you must add the SAN extension to your cert files when generating certs by yourself or the Harbor instance will not start up normally. The `DNS` name in `SAN` exertion should the same as `CN` field in the table below.  For more information please refer to [golang 1.5 release notes](https://golang.org/doc/go1.15#commonname) and [this issue](https://github.com/golang/go/issues/24151).